### DISCUSSION TITLE
unable to remove telegram account

### DISCUSSION THREAD

**Discussion by Dampress** [sourcenetwork/defradb#2328] [General]
I find it difficult to remove the telegram account i mistakenly added. pls work on giving your users the choice to remove accounts they mistakenly added. 
Also, it should not be mandatory for a user to reach a particular level before they can proceed to another task. pls work on that too. 

pls work on the user experience of your site. it looks oversized. 

set the width of your site to 100%

**Comment by fredcarle** [comment]
Hi @Dampress. Thank you for your interest in Open Quest. As noted on the issue you opened, this is the wrong medium to discuss issues related to Open Quest. Please use our Discord server for this.
